src/SFA.DAS.Roatp.CourseManagement.Web.UnitTests/Validators/AddAStandard/SelectAStandardSubmitModelValidatorTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FluentValidation.TestHelper;
+using NUnit.Framework;
+using SFA.DAS.Roatp.CourseManagement.Web.Models.AddAStandard;
+using SFA.DAS.Roatp.CourseManagement.Web.Validators.AddAStandard;
+
+namespace SFA.DAS.Roatp.CourseManagement.Web.UnitTests.Validators.AddAStandard
+{
+    [TestFixture]
+    public class SelectAStandardSubmitModelValidatorTests
+    {
+        [TestCase(0, false)]
```

Comment:
not sure how the 0 is giving an isValid = false here. 0 is not empty...

Reply:
yes zero is considered as empty for non nullable numbers